tests/i18n/patterns/locale/nl/LC_MESSAGES/django.po

```diff
@@ -36,3 +36,7 @@ msgstr "^profiel/"
 #: urls/namespace.py:9 urls/wrong_namespace.py:10
 msgid "^register/$"
 msgstr "^registeren/$"
+
+#: urls/namespace.py:12
+msgid "register-as-path/"
+msgstr "registeren-als-pad/"
```

Comment:
I'm Dutch. I know this should read `registreren-als-pad` instead of `registeren-als-pad` (there should be an `r` after the `t`). However, I decided to be foolishly consistent.

Reply:
Please fix it everywhere.